src/site/stages/build/page-builder/schema-validation.js

```diff
@@ -0,0 +1,39 @@
+const validator = require('./validator');
+const { getContentModelType } = require('./helpers');
+const page = require('./schemas/page');
+
+const schemas = {
+  page,
+};
```

Comment:
There will likely be a lot of schemas, so it might be useful to use a helper function that does a dynamic `require` under-the-hood to import the schema.